### PR TITLE
Adding ontap_firewall_policy.py Module

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
Adding ontap_firewall_policy.py Module

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME
- na_ontap_firewall_policy.py

##### ANSIBLE VERSION

```
bash-3.2# ansible --version
ansible 2.7.0.dev0 (sf_commonfiles caf71e24dd) last updated 2018/08/06 10:46:57 (GMT -700)
  config file = None
  configured module search path = [u'/Users/chrisarchibald/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /private/tmp/ansible/lib/ansible
  executable location = /private/tmp/ansible/bin/ansible
  python version = 2.7.12 (default, Oct 11 2016, 05:20:59) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.38)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by schmots1 — APPROVED** [review]
shipit

File: lib/ansible/modules/storage/netapp/na_ontap_firewall_policy.py [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+    description:
+      - The node to run the firewall configuration on
+    required: True
+'''
+
+EXAMPLES = """
+    - name: create firewall Policy
+      na_ontap_firewall_policy:
+        state: present
+        allow_list: [1.2.3.4/24,1.3.3.4/24]
```
Comment (gundalow) [inline lib/ansible/modules/storage/netapp/na_ontap_firewall_policy.py:64]: Not sure what you've done in other modules, though allowing this to take `type=list` maybe nicer

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/storage/netapp/na_ontap_firewall_policy.py (hunk 1 above)
Comment (lonico) [reply lib/ansible/modules/storage/netapp/na_ontap_firewall_policy.py:64]: It is defined as a list:
allow_list=dict(required=False, type="list"),

Or do you mean the example can be simplified?

Comment (gundalow) [reply lib/ansible/modules/storage/netapp/na_ontap_firewall_policy.py:64]: Oh, ignore this

**Review by gundalow — APPROVED** [review]

**Pull request merged**